tests/serialize/runstate/dynamodb_state_store_test.py

```diff
@@ -263,11 +263,36 @@ def test_delete_item(self, store, small_object, large_object):
         pairs = list(zip(keys, (value for i in range(len(keys)))))
         store.save(pairs)
 
+        store._consume_save_queue()
         for key, value in pairs:
```

Comment:
ignorable nit:
```suggestion
        store.save(pairs)
        store._consume_save_queue()
        
        for key, value in pairs:
```
would be more consistent with the previous test (and make it clearer that the save()/_consume_save_queue() are coupled